src/trix/elements/trix_editor_element.coffee

```diff
@@ -24,6 +24,13 @@ Trix.registerElement "trix-editor", do ->
     return if element.hasAttribute("role")
     element.setAttribute("role", "textbox")
 
+  ensureAriaLabel = (element) ->
+    return if element.hasAttribute("aria-label") or element.hasAttribute("aria-labelledby")
+    return if element.labels.length == 0 or element.labels.some (label) -> label.contains(element)
```

Comment:
texts = (label.textContent for label in element.labels when not label.contains(element))